### PR TITLE
Updated readme file

### PR DESCRIPTION
**Pull request by ipopescu**
### Related links

Requirements card: https://github.com/casper-network/docs/issues/183

### Changes

- Updated the main README file:
    - removed references to the docs-app directory (the old name)
    - the source/docs/casper directory is already mentioned, so I pointed to the next section
    - specified Node version 14+ (instead of npm version 14); npm comes with the Node.js installation
- Removed an old README file in another directory

### Notes

I ran the site locally as well, even though the readme changes shouldn't have had any impact.
I cannot see any precomit hooks failing, but I reached out to the team to follow up on it.

### CODE REVIEW

**Comment by ipopescu** [comment]
After talking with @ACStoneCL, I realized that we should explain how to fork the repository, clone the forked repository and create a PR from the fork. I issued a new commit 9f6a60906de61a383ae8f2e59d95e33feb8b9cca for these steps.

File: README.md [hunk 1]
```diff
@@ -21,21 +20,22 @@ Follow these steps to run the documentation website locally, displayed in your l
 
 ### Running the website locally
 
-1. Clone this repository on your machine: [docs-app](https://github.com/casper-network/casper-node/docs-app).
-2. Open the `docs-app` directory.
-3. Run the following commands:
+1. Create a fork of the documentation repository: https://github.com/casper-network/docs/.
+2. Clone the fork on your machine.
+2. In the root folder, run the following commands:
```
Comment (Buddhika-Pathirathna) [inline README.md:25]: Is it better to mention 'In the forked folder' rather than 'root folder' ?

**Review by Buddhika-Pathirathna — COMMENTED** [review]
Added a comment

File: README.md (hunk 1 above)
Comment (ipopescu) [reply README.md]: Updated with d557e3c.

**Review by Buddhika-Pathirathna — APPROVED** [review]

**Pull request merged**